Gemfile.lock

```diff
@@ -557,11 +557,6 @@ GEM
       errbase (>= 0.1.1)
     safety_net_attestation (0.4.0)
       jwt (~> 2.0)
-    sass (3.7.4)
-      sass-listen (~> 4.0.0)
-    sass-listen (4.0.0)
-      rb-fsevent (~> 0.9, >= 0.9.4)
-      rb-inotify (~> 0.9, >= 0.9.7)
```

Comment:
YESSSSS next up remove `sassc`!!! 🔥 🔥 🔥

Reply:
I don't know if my webpack skills are sufficient for the sprocket replacement journey, but I agree 😛

Reply:
Also guessing we'll want to do the webpack 4 -> webpack 5 thing at some point too

Reply:
>Also guessing we'll want to do the webpack 4 -> webpack 5 thing at some point too

Still waiting on Webpacker v6 😩  ([rc6](https://github.com/rails/webpacker/releases/tag/v6.0.0.rc.6) as of a few weeks ago)